demos/default/pages/getStaticProps/with-revalidate.js

```diff
@@ -25,7 +25,7 @@ export async function getStaticProps(context) {
     props: {
       show: data,
     },
-    revalidate: 1,
+    revalidate: 60,
```

Comment:
I popped a note in the readme because there are various different demo pages with `revalidate: 60` and thought it best not to duplicate the comment everywhere

Reply:
@ascorbic ah, i see - i'll change it back and add a comment